src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -1204,28 +1201,39 @@ private static void PartialMethodChecks(SourceOrdinaryMethodSymbol definition, S
             }
         }
 
-        /// <summary>
-        /// Returns true if the two partial methods have the same constraints.
-        /// </summary>
-        private static bool HaveSameConstraints(SourceOrdinaryMethodSymbol part1, SourceOrdinaryMethodSymbol part2)
+        private static void PartialMethodConstraintsChecks(SourceOrdinaryMethodSymbol definition, SourceOrdinaryMethodSymbol implementation, DiagnosticBag diagnostics)
```

Comment:
Replace comment with a new description?

Reply:
> Replace comment with a new description?

I think the new name is sufficiently self-descriptive. It does what it does and it doesn't return any information that a caller has to understand and react to.

---
In reply to: [279126526](https://github.com/dotnet/roslyn/pull/35272#discussion_r279126526) [](ancestors = 279126526)